python/dgl/graphbolt/impl/ondisk_dataset.py

```diff
@@ -38,6 +41,215 @@
 __all__ = ["OnDiskDataset", "preprocess_ondisk_dataset", "BuiltinDataset"]
 
 
+def _graph_data_to_fused_csc_sampling_graph(
```

Comment:
is it possible to split this func into smaller ones?

Reply:
I'm afraid not.